rfd/0138/README.md

```diff
@@ -0,0 +1,146 @@
+
+```
+            DHCP Server                  DHCP Relay                  Booting CN
+                +                            +                            +
+                |                            |       DHCPDISCOVER         |
+                |                            <----------------------------+
+                |  DHCPDISCOVER with NICTAG  |                            |
+                <----------------------------+                            |
+                |                            |                            |
++---------+     |                            |                            |
+|         <-----+                            |                            |
+|  NAPI   |     |                            |                            |
+|         +----->                            |                            |
++---------+     |   DHCPOFFER with NICTAG    |                            |
+                +---------------------------->                            |
+                |                            |         DHCPOFFER          |
+                |                            +---------------------------->
+                |                            |                            |
+                |                            |                            |
+                |                            |                            |
+                +                            +                            +
+
+``` 
+
+## Multi-subnet Admin Network Deployment Considerations
+
+_many of these could be implemented as future work_
+
+* When a new admin subnet is deployed the HN, and any other CNs in on different subnets will need new static routes to the new admin subnet before any of the CNs on the new subnet are booted.
+* The NTP server's config file, `/etc/inet/ntp.conf`, will need to be modified to allow other admin subnets to query it.  Example:
+```
+restrict 10.99.99.0 mask 255.255.255.0
+restrict 10.222.222.0 mask 255.255.255.0
+```
+* Apply firewall rules to allow traffic to core services from the new rack(subnet).  Example:
```

Comment:
You can reference RFD 117 as a possible solution here, since we can then write:

```
FROM network tag "admin" TO tag "smartdc_role" ALLOW tcp PORT all
```